commands_control.py

```diff
@@ -12,10 +12,10 @@
 # to allow it to work without a space following it
 class Control_Comment(command_base.Command_Text_Basic):
     def __init__(
-        self, 
+        self,
         ):
 
-        super().__init__("-",  # the name of the command as you have to enter it in the code
+        super().__init__("-, Comment",

```

Comment:
I don't think that we need a command for this, it's probably faster to just ignore any lines that start with `-`

Reply:
That's the purpose of it :-)